### ISSUE TITLE
[508-defect-3] CST: link file instructions to add file button

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#41814]
## Description

Add file instructions / descriptions should be semantically linked to the add file button using `aria-describedby`

Related: https://github.com/department-of-veterans-affairs/va.gov-team/issues/41447 (under "Should" fix)

![The accepted file type and maximum file size information needs to be associated to the add file button](https://user-images.githubusercontent.com/14154792/169866025-f86d87a3-452f-4dae-a5bb-d239cae19268.PNG)

## Tasks

- [ ] Add `aria-describedby` attribute to add file button
- [ ] Add tests

## Definition of done

- [ ] All tasks complete
- [ ] All tests passing
